scheduler/roles/k8s_start_services/vars/main.yml

```diff
@@ -37,7 +37,7 @@ gpu_discovery_feature: true
 xilinx_device_plugin_yaml_url: "{{ hostvars['localhost']['offline_manifest_path'] }}/xilinx-device-plugin.yaml"
 spark_operator_repo: "{{ hostvars['localhost']['offline_tarball_path'] }}/spark-operator-v1beta2-1.3.8-3.1.1.tar.gz"
 rocm_device_plugin_yaml_url: "{{ hostvars['localhost']['offline_manifest_path'] }}/rocm-device-plugin.yaml"
-habana_device_plugin_yaml_url: "{{ hostvars['localhost']['offline_manifest_path'] }}/habana-k8s-device-plugin.yaml"
+habana_device_plugin_yaml_url: "{{ hostvars['localhost']['offline_manifest_path'] }}/habana-device-plugin.yaml"
```

Comment:
This file name change to be reverted. By changing the file name, it fails to deploy the pod.

Reply:
local_repo.yml will rename the file as `package_name`, refer to here https://github.com/dell/omnia/blob/release_1.7/local_repo/roles/parse_and_download/files/download_common.py#L196

The `package_name` of device plugin is `habana-device-plugin`, https://github.com/dell/omnia/blob/release_1.7/input/config/ubuntu/22.04/k8s.json#L95

Can you please check the device plugin file name under `/omnia_repo/cluster/manifest/`, or remove the exist habana device plugin yaml file and run `local_repo.yml` again to see what's the file name? In our environment the file name is `habana-device-plugin.yaml` and we can deploy the pods.